applications/nrf5340_audio/README.rst

```diff
@@ -1253,6 +1256,8 @@ Testing the BIS mode is identical to `Testing the default CIS mode`_, except for
 * Pressing the **PLAY/PAUSE** button on the gateway will respectively start or stop the stream for all headsets listening in.
 * Pressing the **BTN 4** button on a headset will change the active audio stream.
   The default configuration of the BIS mode supports two audio streams (left and right).
+* Pressing the **BTN 5** button on a headset will change the gateway source for the audio stream.
+  The default configuration of the BIS mode supports two gateways (``CONFIG_BT_DEVICE_NAME`` and ``CONFIG_BT_DEVICE_NAME_ALT``).
```

Comment:
We should probably add a note here saying that we must build firmware and program a gateway twice, one time "per gateway name"